lib/awful/tag.lua

```diff
@@ -471,7 +486,21 @@ end
 -- @param add Value to add to number of column windows.
 -- @param t The tag to modify, if null tag.selected() is used.
 function tag.incncol(add, t)
-    tag.setncol(tag.getncol(t) + add, t)
+    client = client or require("awful.client")
+    local ntiled = #client.tiled(1)
```

Comment:
That should be the tag's screen probably (assuming that `1` is the screen here).